pages/guides/docker_containerized_builds.md.erb

```diff
@@ -45,12 +45,12 @@ app:
     - ./:/go/src/github.com/buildkite/agent
     - /usr/bin/buildkite-agent:/usr/bin/buildkite-agent
   environment:
-    - BUILDKITE_AGENT_ACCESS_TOKEN
-    - BUILDKITE_JOB_ID
-    - BUILDKITE_BUILD_ID
-    - BUILDKITE_BUILD_NUMBER
-    - GITHUB_RELEASE_ACCESS_TOKEN
-    - DOCKER_HUB_TRIGGER_TOKEN
+    - BUILDKITE_AGENT_ACCESS_TOKEN: $BUILDKITE_AGENT_ACCESS_TOKEN
```

Comment:
Hey @daveslutzkin, thanks for making these changes! I remember not needing to define the extra `: $...` part of a ENV if you just wanted the value to be copied to the container. Just define the name of the ENV and docker compose would do the rest, did they deprecate that functionality?